pkg/autoscaling/handler.go

```diff
@@ -50,12 +52,30 @@ func (h *HTTPHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	log.Debug("autoscaling: get http body completed", zap.String("strategy", string(data)))
```

Comment:
```suggestion
	log.Debug("get http body completed", zap.String("strategy", string(data)))
```
I think `autoscaling:` is unnecessary

Reply:
done